WorkspaceServer.Tests/Kernel/LanguageKernelTests.cs

```diff
@@ -943,28 +944,33 @@ public async Task When_SubmitCode_command_adds_packages_to_csharp_kernel_then_th
                 .NotContain(e => e.Code.Contains("#r"));
         }
 
-        [Fact]
-        public async Task When_SubmitCode_command_adds_packages_to_csharp_kernel_then_PackageAdded_event_is_raised()
+        [Theory]
+        [InlineData(Language.CSharp, "Microsoft.Extensions.Logging.ILogger logger = null;")]
+        [InlineData(Language.FSharp, "let logger: Microsoft.Extensions.Logging.ILogger = null")]
+        public async Task When_SubmitCode_command_adds_packages_to_kernel_then_PackageAdded_event_is_raised(Language language, string expression)
         {
-            var kernel = new CompositeKernel
+            IKernel kernel = language switch
```

Comment:
Can use var

Reply:
Can't because the expression is inferred to be of type `CSharpKernel` which makes the F# line fail type inference.  If I kept var I would have to add `as IKernel` after the C# one so I instead opted to type up front.